al-khaser/Anti Debug/NtQueryObject_AllTypesInformation.cpp

```diff
@@ -10,6 +10,8 @@ locate "DebugObject". Todo: Support for Win10
 
 BOOL NtQueryObject_ObjectAllTypesInformation()
 {
+	//NOTE this check is unreliable, a debugger present on the system doesn't mean it's attached to you
+

```

Comment:
I mean by *Same goes for NtQueryObject_ObjectTypeInformation()* that I am thankful for comment and fixes. Going back to *NtQueryObject_ObjectTypeInformation* method, why can't we say that if ObjectInformation->TotalNumberOfObjects == 1, means no debugger and everything else, something fishy either a hook or debugger ?